ocs_ci/templates/workloads/pgsql/server/StatefulSet.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: postgres
+  namespace: my-ripsaw
+spec:
+  serviceName: "postgres"
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: postgres
+        role: postgres
+    spec:
+      containers:
+        - name: postgres
+          image: postgres:10.4
```

Comment:
10.5 image seems to be available. why not use postgres:latest instead?

Reply:
I haven't tried 10.5 image.  Will look into this.

Reply:
we can take this up in next testing.